sdks/java/io/bigquery-io-perf-tests/build.gradle

```diff
@@ -17,15 +17,15 @@
  */
 
 plugins { id 'org.apache.beam.module' }
-applyJavaNature(exportJavadoc: false, publish: false)
+applyJavaNature(enableStrictDependencies:true,exportJavadoc: false, publish: false)
 provideIntegrationTestingDependencies()
 enableJavaPerformanceTesting()
 
 description = "Apache Beam :: SDKs :: Java :: Google BigQuery IO Performance tests"
 ext.summary = "Performance tests for Google BigQuery IO sources and sinks"
 
 dependencies {
-    compile library.java.google_api_services_bigquery
+    testImplementation library.java.google_api_services_bigquery
```

Comment:
No need for compile+permitUnusedDeclared here since this bigquery-io-perf-tests just contains tests.